### ISSUE TITLE
Suggestion on Presbyopia Correction for Windows Mixed-Reality Enthusiasts Guide

### ISSUE THREAD

**Issue by j-a-lewis** [MicrosoftDocs/mixed-reality#156]
I'm interested in getting an HP Reverb G2 for Microsoft Flight Simulator 2020.  I understand there is no "eye relief" for the G2 or other Windows Mixed-Reality headsets, at least according to the specs for the standard.  The HP Reverb G2 FAQs section in the Enthusiast's Guide essentially just says, "Just wear your corrective lens."  But the guide and the standard don't address the question of presbyopia, i.e., age-related loss of close-up vision due to hardening of the eye lens.  Normally, presbyopia correction is only applied to the lower part of the visual field in prescription glasses or contact lenses (think bifocals).  Thus, typical presbyopia correction in prescriptive eyewear will NOT work to see the entire visual field in a headset clearly.  What might work, for folks that don't have significant astigmatism as well, is to wear full-lens drug-store OTC reading glasses, short of having custom prescription lens inserts made for a set of goggles.  What would very much help an older prospective headset user like me make an intelligent purchase of full-frame reading glasses or make sure I'm having the right prescription lens inserts made is to reveal in the Enthusiast's Guide the effective focal distance that the virtual screen images for each eye are displayed in the headset.  The amount of +OD correction drops as the virtual screen display image distance moves farther away from 1 m to 2, 3, or more meters.  So what would really help for your table of specs on Windows Mixed-Reality headsets is to add a column for each model headset and provide the out-of-the-box distance that each model of headset is set to display its virtual screen display images from the wearer's eyes. The OEM's should be able to provide these figures and there is no point keeping it "secret" as they do.  The figure ought to be in the headset specs of devices, too, for the reasons I state above.  Obviously, if one is seriously near-sighted or far-sighted, the situation is more complicated than that.  But I've had cataract surgery, I've had "perfect" monofocal plastic distance vision lenses inserted in each eye and my only optical problem is the requirement for presbyopia correction, which varies according to the relative near distance of the image my eyes are trying to see.  The varying requirements for +OD correction according to the distance of an image for an eye in presbyopia are shown in the following table:
![image](https://user-images.githubusercontent.com/41112084/103426075-5cce8b80-4b7c-11eb-8374-ceeb2506a42e.png)
taken from the following blog post on figuring out reading glasses needs based on one's prescription: https://contactsadvice.com/2019/10/how-to-determine-the-strength-of-reading-glasses-you-need/

TL;DR Microsoft in its Enthusiast's Guide for Mixed-Reality headsets should address the issue of presbyopia if it wants to encourage folks over 40 to be interested in mixed-reality.  It would be great if more provision and advice were provided for applying correction out of the box to Windows Mixed-Reality headsets rather than leaving the users, especially older folks like me, at a loss as to what to do.  Apple is said to be working on visual field correction built into any VR/AR headsets it produces.  Would be great if Microsoft would start thinking about the same.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96c577f1-090c-caed-e039-20625d17485d
* Version Independent ID: 7d9d3ae8-6945-ddea-ac0e-eae8397ecb25
* Content: [HP Reverb G2 FAQs - Enthusiast Guide](https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/reverbg2-faq#what-should-i-do-to-get-a-crisper-image)
* Content Source: [enthusiast-guide/reverbG2-faq.md](https://github.com/MicrosoftDocs/mixed-reality/blob/docs/enthusiast-guide/reverbG2-faq.md)
* Product: **mixed-reality**
* Technology: **enthusiast-guide**
* GitHub Login: @hferrone
* Microsoft Alias: **v-hferrone**

**Comment by hferrone** [comment]
Great point @j-a-lewis! 

@tesych @tmlyon Can you confirm/assist?

**Comment by hferrone** [comment]
Hi @j-a-lewis - Since this isn't a developer issue, but a hardware/support topic, we can't advise or add this kind of information to our docs. This would be something that HP, or any headset manufacturer, would be responsible for. Thanks!

**Comment by j-a-lewis** [comment]
As you think best.  However, the Windows Mixed-Reality Enthusiast's Guide does have a table of hardware specs, including IPD distances, and in the HP Reverb G2 FAQ, the issue of how to get a crisper image is discussed, clearly a hardware-related image (or where does software come into that?).  Presumably, the point of the Guide is to have members of the general public have a favorable experience with Windows Mixed-Reality and treating a particular and almost universal vision problem that occurs as folks age past about 40 years as a non-issue for an Enthusiast's Guide is not helpful, IMHO, whereas with the resources available to Microsoft, it should not be hard to come up with some helpful suggestions that would apply across all HMD headset (if we only knew what the focal distance is for the virtual images).  I hope as passes those who support Windows Mixed-Reality Enthusiasts will come to be people more supportive and creative in finding solutions to common problems that might occur with Windows Mixed-Reality headset.
